pkg/authorization/apis/authorization/helpers.go

```diff
@@ -100,6 +101,26 @@ func BuildSubjects(users, groups []string, userNameValidator, groupNameValidator
 	return subjects
 }
 
+func BuildRBACSubjects(users, groups []string) []rbac.Subject {
```

Comment:
This is only used in one spot.  move it to the point of use to prevent proliferation.

Reply:
@deads2k I moved the function and made it private.

Reply:
See https://github.com/openshift/origin/pull/16034#discussion_r135891136